src/inc/quic_storage.h

```diff
@@ -34,6 +34,13 @@ void
 
 typedef CXPLAT_STORAGE_CHANGE_CALLBACK *CXPLAT_STORAGE_CHANGE_CALLBACK_HANDLER;
 
+typedef enum CXPLAT_STORAGE_OPEN_FLAGS {
+    CXPLAT_STORAGE_OPEN_FLAG_NONE =     0x0,
+    CXPLAT_STORAGE_OPEN_FLAG_WRITABLE = 0x1
```

Comment:
Should we make a READABLE one too, for the sake of completeness?

Reply:
I don't think so; What use would a non-readable, non-writable Storage object be (it would be an error)?  Or a write-only one?

Reply:
I was just thinking the user might want write only or read only. I don't feel strongly about it right now though.

Reply:
I made the default flag (0) represent READABLE, so it's always READABLE, and WRITEABLE and DELETEABLE are optional.